subprojects/internal-integ-testing/src/main/groovy/org/gradle/integtests/fixtures/AbstractIntegrationTest.java

```diff
@@ -145,15 +145,31 @@ public static String mavenCentralRepository() {
     }
 
     public void useRepositoryMirrors() {
+        useRepositoryMirrors(false);
+    }
+
+    public void useRepositoryMirrors(final boolean global) {
```

Comment:
I think we should have the code in one place, now that it is getting more complicated, and call it from Groovy and Java.

Reply:
Moved to `GradleExecuter` so groovy and java code can both reference it now.